### PR TITLE
Added CreatePublicationHandler

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: create-publication/build.gradle [hunk 1]
```diff
@@ -0,0 +1,11 @@
+repositories {
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
```
Comment (axthosarouris) [inline create-publication/build.gradle:2]: A comment what is this repository for?

File: modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java [hunk 2]
```diff
@@ -19,6 +17,8 @@
 import org.apache.http.HttpStatus;
 import org.slf4j.LoggerFactory;
 
+import static nva.commons.utils.JsonUtils.objectMapper;
```
Comment (axthosarouris) [inline modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java:20]: We should decide if we want the static imports to be above or under the non-static imports

File: publication-commons/src/main/java/no/unit/nva/publication/JsonLdContextUtil.java [hunk 3]
```diff
@@ -2,13 +2,14 @@
...
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.io.InputStream;
-import java.nio.file.Path;
-import java.util.Optional;
 import nva.commons.utils.IoUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.InputStream;
```
Comment (axthosarouris) [inline publication-commons/src/main/java/no/unit/nva/publication/JsonLdContextUtil.java:9]: run an "Organize imports" on the whole project

**Comment by axthosarouris** [comment]
Run an "organize imports" to see if we have different settings.

File: modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java (hunk 2 above)
Comment (janbj) [reply modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java:20]: This is intelliJ Optimize Imports

File: publication-commons/src/main/java/no/unit/nva/publication/JsonLdContextUtil.java (hunk 3 above)
Comment (janbj) [reply publication-commons/src/main/java/no/unit/nva/publication/JsonLdContextUtil.java:9]: I did.

File: create-publication/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+        GatewayResponse<JsonNode> expected = new GatewayResponse<>(
+                handler.toJsonNodeWithContext(publication),
+                TestHeaders.getResponseHeaders(),
+                HttpStatus.SC_CREATED
+        );
+
+        Assertions.assertEquals(expected, actual);
+    }
+
```
Comment (brinxmat) [inline create-publication/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java:78]: Test for bad input

**Pull request merged**